src/vite-env.d.ts

```diff
@@ -1,4 +1,16 @@
 /// <reference types="vite/client" />
 
-declare module '@agoric/web-components/react';
-declare module '@agoric/notifier';
+declare module '@agoric/web-components/react' {
+  export const makeReactAgoricWalletConnection;
+}
+
+declare module '@agoric/notifier' {
+  export const makeAsyncIterableFromNotifier;
+}
+
+declare module '@agoric/casting' {
+  export type Leader = any;
+  export const makeFollower;
+  export const iterateLatest;
+  export const makeLeader;
```

Comment:
these aren't as useful as exporting types because they will have symbols as you use them.

Reply:
If I don't include these I get `Module '"@agoric/casting"' has no exported member 'iterateLatest'.` etc.

Reply:
ahh, I didn't think about how once we add a block here to (define `Leader`) we lose the "shorthand" feature and have to write out all of them.

thankfully there won't be a lot to import from casting.